dbt_coves/tasks/extract/airbyte.py

```diff
@@ -76,13 +79,12 @@ def run(self):
         self.airbyte_api_caller = AirbyteApiCaller(airbyte_host, airbyte_port)
 
         console.print(
-            f"Extracting Airbyte's [b]Source[/b], [b]Destination[/b] and [b]Connection[/b] configurations to {os.path.abspath(path)}\n"
+            f"Extracting Airbyte's [b]Source[/b], [b]Destination[/b] and [b]Connection[/b] configurations to {os.path.abspath(extract_destination)}\n"
         )
 
-        dbt_sources_list = shell.run_dbt_ls(
-            "dbt ls --resource-type source",
-            None,
-        )
+        dbt_ls_cmd = f"dbt ls --resource-type source {dbt_modifiers}"
```

Comment:
if `dbt_modifiers` is `None`, that string would be "dbt ls --resource-type source None"?

Reply:
it seems it isn't

I've tested letting **dbt_list_args** be defaulted to None (by removing it from .dbt_coves.yml config file and not using it in CLI) and dbt_ls_cmd string is correct

```
>>dbt_ls_cmd
'dbt ls --resource-type source '
```